### ISSUE TITLE
[Bug]: Server hang/crash

### ISSUE THREAD

**Issue by ukmcplyr** [FTBTeam/FTB-Modpack-Issues#817]
### Modpack

FTB Presents Direwolf20 1.18

### Modpack version

1.2.0

### Log Files

logs/latest.log:
- https://pste.ch/evakelekon.sql

logs/debug.log:
- https://pste.ch/qezayuxeyi.md

Crash log:
- https://pste.ch/odihutituf.rb

### Describe the bug

Light on details as the server was idle with no players connected, apologies.

Stuff that would have been active:

- Minimal RS (1 64k disk, 1 drive, 2 x external storage hooked up to two Function Storage controllers and a bunch of drawers)
- 2 x Mekanism advanced solar generators and 1 windmill
- 2 x tree farms and one wheat farm (all create-based)
- Small Laser IO setup distributing items between chests using a couple of relays
- A bunch of Mekanism Quantum Entangloporters distributing items and power

I've probably missed something important but hopefully there's enough context in the logs linked. Give me a shout if there's anything else I can provide, I'll keep the state of the server post-crash around just in case.

### Steps to reproduce

N/A

### Expected behaviour

Less crashing :)

**Comment by Sunekaer** [comment]
Do you have a lot of mobs or something in your world?

**Comment by ukmcplyr** [comment]
Great question. There was a passive mob farm running which I forgot to mention which was very slowly chewing its way through them. I don't think there was an excessive number when I logged back in but I'll double check and post back.

I have just noticed something strange within the past hour. 117 enderman lingering in roughly the same area one hundred blocks or so away from where I've been idling. Moving about a bit in spectator mode to investigate and that number dropped to ~90 but is going up again at the rate of one enerman every ~5 minutes. Unsure if related?
